### DISCUSSION TITLE
My Idea

### DISCUSSION THREAD

**Discussion by Aiz08** [skyline-emu/skyline#178] [Ideas]
why Don't we Copy the Code/file for Egg Ns Emulator To Run the Roms Is It Legal?

**Comment by leopeo-star** [comment]
It was supposed to be open source right couldn't like dev just copy egg ns wouldn't it just speed up the development i mean they also copied it in the right place and made it paid right

**Reply by Aiz08** [reply]
i know right The Layout/Controls Of Of Switch Is There It Only Need The File For Running The Roms Then Its Perfect

**Comment by bylaws** [comment]
This isn't possible, eggns stole code.

Please avoid asking here in the future, use other mediums of communication for suggestions rather than github
